lib/ClassMover/Tests/Adapter/TolerantParser/TolerantClassReplacerTest.php

```diff
@@ -38,7 +38,8 @@ public function testFind($fileName, $classFqn, $replaceWithFqn, $expectedSource)
         self::assertStringContainsString($stripEmptyLines($expectedSource), $stripEmptyLines($edits->apply($source->__toString())));
     }
 
-    public function provideTestFind()
+    /** @return array<array<string>> */
```

Comment:
not for future, prefer `list<string>` where it makes sense.

Reply:
I have never understood what the difference between those is. I mean the most generic would be `iterable<string>` what would be the difference there?

Reply:
list is an array, but when used like a vector: `[1,2,3,4]`, i.e. the keys are implicit and correspond to the index of the value.

Reply:
But in this case we have explicit string keys.